lib/exercism/team_stream_filters.rb

```diff
@@ -72,7 +72,7 @@ def sql
     # rubocop:disable Metrics/MethodLength
     def views_sql
       <<-SQL
-      SELECT 1 AS id, COUNT(views.id) AS total
+      SELECT 'team_stream' AS id, COUNT(1) AS total
```

Comment:
Yeah, that sounds like a great approach. We definitely don't need the stream name.